packages/desktop/views/onboarding/views/create-from-ledger/views/ledger-connection-guide/LedgerConnectionGuideView.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+    import { OnboardingLayout } from '@views/components'
+    import { localize } from '@core/i18n'
+    import { LedgerAppOpen, LedgerDeviceUnlocked, LedgerLiveNotOpen, LedgerStillNotConnected } from './components'
+    import { createFromLedgerRouter } from '../../create-from-ledger-router'
+
+    let stepIndex: number = 0
+
+    const LEDGER_CONNECTION_GUIDE_STEPS: unknown[] = [
```

Comment:
No need for these kind of typings, TS will automatically defer the type.
```suggestion
    const LEDGER_CONNECTION_GUIDE_STEPS = [
```